rockstar/rockstar.js

```diff
@@ -1138,12 +1338,13 @@
         function toggleSide() {
             popup.toggleClass('rs_toggle');
         }
-        const popup = $(`<div style='position: absolute; z-index: 1000; top: 0px; max-height: calc(100% - 28px); max-width: calc(100% - 28px); padding: 8px; margin: 4px; overflow: auto; ` +
-                `background-color: white; border: 1px solid #ddd;'>` +
+        const popup = $(`<div class="rockstarPopup">` +
                 `${main ? "<span class=title><span style='font-size: 18px;'>≡</span> " : "<span style='font-weight: bold'>"}${title}</span>` +
-                `<div style='display: block; float: right;'>${main ? "<span class=toggleSide style='padding: 4px'> ⇄ </span><span class=minimize style='padding: 4px'> _ </span>" : ""} ` + 
-                `<a href='https://gabrielsroka.github.io/rockstar/' target='_blank' rel='noopener' style='font-size: 18px; padding: 4px'>?</a> ` + 
-                `<a onclick='document.body.removeChild(this.parentNode.parentNode)' style='font-size: 18px; cursor: pointer; padding: 4px'>X</a></div><br><br></div>`)
+                `<div class='rockstarButtons' style='display: block; float: right;'>${main ? "<span class=toggleSide style='padding: 4px'> ⇄ </span><span class=minimize style='padding: 4px'> _ </span>" : ""} ` + 
+                (main ? `<a class='newsButton'><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-wclassth="1.5" stroke="currentColor"><path stroke-linecap="round" stroke-linejoin="round" d="M21 11.25v8.25a1.5 1.5 0 0 1-1.5 1.5H5.25a1.5 1.5 0 0 1-1.5-1.5v-8.25M12 4.875A2.625 2.625 0 1 0 9.375 7.5H12m0-2.625V7.5m0-2.625A2.625 2.625 0 1 1 14.625 7.5H12m0 0V21m-8.625-9.75h18c.621 0 1.125-.504 1.125-1.125v-1.5c0-.621-.504-1.125-1.125-1.125h-18c-.621 0-1.125.504-1.125 1.125v1.5c0 .621.504 1.125 1.125 1.125Z" /></svg></a>` : '') +

```

Comment:
use emoji instead of svg 🎁⚙️

Reply:
Yes that's what I had initially done but then it always shows as this full-color emoji and thought the minimalist outline of the SVGs looked nicer (taken from https://heroicons.com/). What do you think? You prefer the color emojis?

# Emojis

![image](https://github.com/gabrielsroka/gabrielsroka.github.io/assets/2124035/8d9f0889-c970-45df-b40b-88349844cbb9)

# SVGs

![image](https://github.com/gabrielsroka/gabrielsroka.github.io/assets/2124035/a2bc235d-e4b7-48ac-aab8-5da3d5469017)